test/unit/lib/gui/tool-runner/index.js

```diff
@@ -40,44 +40,44 @@ describe('lib/gui/tool-runner/index', () => {
         }));
     };
 
-    const mkToolCliOpts_ = (globalCliOpts = {name: () => 'hermione'}, guiCliOpts = {}) => {
+    const mkToolCliOpts_ = (globalCliOpts = {name: () => 'testplane'}, guiCliOpts = {}) => {
         return {program: globalCliOpts, options: guiCliOpts};
     };
     const mkPluginConfig_ = (config = {}) => {
         const pluginConfig = _.defaults(config, {path: 'default-path'});
         return {pluginConfig};
     };
 
-    const mkHermione_ = (config, testsTree) => {
-        const hermione = stubTool(config, {UPDATE_REFERENCE: 'updateReference'});
-        sandbox.stub(hermione, 'emit');
-        hermione.readTests.resolves(mkTestCollection_(testsTree));
+    const mkTestplane_ = (config, testsTree) => {
+        const testplane = stubTool(config, {UPDATE_REFERENCE: 'updateReference'});
+        sandbox.stub(testplane, 'emit');
+        testplane.readTests.resolves(mkTestCollection_(testsTree));
 
-        return hermione;
+        return testplane;
     };
 
-    const initGuiReporter = (hermione, opts = {}) => {
+    const initGuiReporter = (testplane, opts = {}) => {
         opts = _.defaults(opts, {
             paths: [],
             configs: {}
         });
 
         const configs = _.defaults(opts.configs, mkToolCliOpts_(), mkPluginConfig_());
 
-        return ToolGuiReporter.create(opts.paths, hermione, configs);
+        return ToolGuiReporter.create(opts.paths, testplane, configs);
     };
 
     beforeEach(() => {
-        hermione = stubTool();
+        testplane = stubTool();
 
-        createTestRunner = sinon.stub();
+        createTestRunner = sandbox.stub();
```

Comment:
replaced `sinon` with `sandbox` in the file, because some other tests (in another files) were failing without it